src/controller/java/templates/ChipStructs-java.zapt

```diff
@@ -0,0 +1,58 @@
+{{> header}}
+{{#if (chip_has_client_clusters)}}
+package chip.devicecontroller;
+
+import androidx.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Optional;
+
+public class ChipStructs {
+
+{{#zcl_clusters}}
```

Comment:
This iterates over all clusters where as the above if seems to hint at only client clusters. Is the "all clusters" correct?

Is zcl_clusters correct or should we use chip_clusters? Is there a difference in enabling between the two?